src/custom/state/claim/hooks/index.ts

```diff
@@ -966,7 +966,7 @@ function _enhanceClaimData(claim: UserClaimData, chainId: SupportedChainId, pric
 
   // Free claims will have tokenAndAmount === undefined
   // If it's not a free claim, store the price and calculate cost in investment token
-  if (tokenAndAmount?.amount) {
+  if (tokenAndAmount?.amount && Number(tokenAndAmount.amount) > 0) {
```

Comment:
Approving, although i don't get why price would be 0. It is hardcoded in the contract, so unless they haven't been loaded yet, i don't get how can they be zero. 

Can this be hiding some other issue?

Reply:
It seems like a lag between switching from Gnosis Chain, [where USDC price is 0](https://blockscout.com/xdai/mainnet/token/0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB/read-contract)